features/admin.applications.v1/components/forms/general-details-form.tsx

```diff
@@ -210,6 +210,7 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         };
 
         if (isDiscoverable && !values.accessUrl) {
+            console.log("values.accessUrl", values.accessUrl)
```

Comment:
Remove console logs.

Reply:
Hope you have setup eslint plugin (https://github.com/wso2/identity-apps/blob/master/docs/SET_UP_DEV_ENVIRONMENT.md#vs-code). This will complain about such style issues.

Reply:
Fixed with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)